data/kernels/basic.cl

```diff
@@ -33,6 +33,49 @@ letsgofloat_1ui(read_only image2d_t in, write_only image2d_t out,
   write_imagef (out, (int2)(x, y), (float4)(pixel / 65535.0f, 0.0f, 0.0f, 0.0f));
 }
 
+
+int
+BL(const int row, const int col)
```

Comment:
does that break x-trans and esoteric sensors? or would these always use a single black level instead?